plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/file/IcebergFileMetastoreCatalogModule.java

```diff
@@ -33,6 +35,7 @@ protected void setup(Binder binder)
         binder.bind(IcebergTableOperationsProvider.class).to(FileMetastoreTableOperationsProvider.class).in(Scopes.SINGLETON);
         binder.bind(TrinoCatalogFactory.class).to(TrinoHiveCatalogFactory.class).in(Scopes.SINGLETON);
         binder.bind(MetastoreValidator.class).asEagerSingleton();
+        binder.bind(Key.get(boolean.class, HideDeltaLakeTables.class)).toInstance(false);
```

Comment:
Use `TestingIcebergPlugin.HIDE_DELTA_LAKE_TABLES` (of course requires moving the constant)